nekoyume/battle/components/stats.py

```diff
@@ -36,7 +36,7 @@ def calc_hp_max(self):
 
     def damaged(self, damage):
         damage -= self.calc_defense()
-        self.hp = max(self.hp - damage, 0)
+        self.hp -= damage
```

Comment:
Can `hp` be a negative integer now?

Reply:
Yes

Reply:
![](https://i.ytimg.com/vi/_YPMDG0ZqTA/maxresdefault.jpg)